plugins/flytekit-data-fsspec/tests/test_basic_dfs.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+import pyarrow as pa
+from flytekitplugins.fsspec.pandas import get_storage_options
+
+from flytekit import kwtypes, task
+from flytekit.configuration import aws
+
+try:
+    from typing import Annotated
+except ImportError:
+    from typing_extensions import Annotated
+
+
+def test_get_storage_options():
+    uri = "s3://bucket/somewhere"
+    with aws.S3_ENDPOINT.get_patcher(uri):
```

Comment:
this option isn't supposed to be the bucket.  We added this to support minio in local execution.  See https://github.com/flyteorg/flyte/blob/master/deployment/sandbox/flyte_helm_generated.yaml#L303

Reply:
Updated it, thanks.